superset/views/core.py

```diff
@@ -2071,9 +2094,6 @@ def sqllab_viz(self):
         cols = []
         for config in data.get("columns"):
             column_name = config.get("name")
-            SqlaTable = ConnectorRegistry.sources["table"]
```

Comment:
This abstraction seemed necessary given we're restricting the models to the SQLA connector.